Item.h

```diff
@@ -5,13 +5,19 @@
 struct Character;
 struct Item
 {
-    virtual ~Item() { }
-    virtual void use(Character*) = 0;
-    Item(const std::string& name_, int effect ) : name(name_), boost(effect) { }
+    Item(const std::string& name_, int effect );
     
+    virtual ~Item();
+
+    virtual void use(Character*);
```

Comment:
This used to be pure virtual.  Why did you feel the need to change it?
- [ ] put it back.

Reply:
done.